deblur/deblurring.py

```diff
@@ -55,7 +65,31 @@ def get_sequences(input_seqs):
     return seqs
 
 
-def deblur(input_seqs, read_error=0.05, mean_error=0.005, error_dist=None,
+def create_error_profile(read_error):
+    """Create a theoretical read error profile given
+    a maxima error rate
+
+    Parameters
+    ----------
+    read_error : float
+        the upper bound on the hamming one read error
+
+    Results
+    -------
+    error_profile : list of float
+        the predicted theoretical error profile
+        (note for hamming>1 we take the max of
+        read_error^2 and 0.01 since closer to reality)
+    """
+    error_profile = [1.0, read_error, np.max([0.01, np.power(read_error, 2)]),
```

Comment:
following discussion with @mortonjt , we intend to remove the --read-error option and get rid of the create_error_profile() function.
It has no real use (since read errors are not independent - as is assumed in the create_error_profile() function), and it can confuse the users.
This option was used for comparisons to the illumina error profile (to see how different it is from the simple assumptions), and i don't think it should be used anymore.
Just didn't want to complicate too many things on this PR 
so will remove it in a following PR